keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -676,6 +676,26 @@ Other components:
 * check how the change of meaning of resource requests influence other
   Kubernetes components.
 
+### Instrumentation
```

Comment:
do you think it could be useful to have this for 1.31 alpha to have that feedback going into beta?

Reply:
Yes. I think all the changes discussed for beta can go in while it's still in alpha, but I can specifically prioritize this one.